app/javascript/pages/getAssistance/disclaimer.tsx

```diff
@@ -1,12 +1,47 @@
 import React from "react"
 import Layout from "../../layouts/Layout"
 import withAppSetup from "../../layouts/withAppSetup"
+import { ConfigContext } from "../../lib/ConfigContext"
+import { t, PageHeader } from "@bloom-housing/ui-components"
+import { MailingListSignup } from "../../components/MailingListSignup"
 
 const Disclaimer = () => {
+  const { getAssetPath } = React.useContext(ConfigContext)
   return (
-    <Layout title={"Disclaimer"}>
-      {/* Add PageHeader Component (Look in AssistanceLayout) */}
-      <h1>Disclaimer Content</h1>
+    <Layout title={t("pageTitle.disclaimer")}>
+      <PageHeader
+        title={t("pageTitle.disclaimer")}
+        subtitle={t("disclaimer.intro")}
+        inverse
+        backgroundImage={getAssetPath("bg@1200.jpg")}
+      />
+      {
+        <article className="flex flex-wrap relative max-w-5xl m-auto w-full">
+          <div className="w-full md:w-2/3" data-test-id="assistance-main-content">
```

Comment:
Just wondering what `data-test-id` does? We're using the value `"assistance-main-content"` both here and in `AssistanceLayout.tsx`